### PR TITLE
Add details about DeleteEvent

### LINKED ISSUE #78: Add Details about Delete event
Current Condition:

screenshot from 2018-12-11 17-58-30

### PR DESCRIPTION
**Pull request by merty**
Closes #78

### CODE REVIEW

File: git_stalk/stalk.py [hunk 1]
```diff
@@ -88,6 +90,7 @@ def get_details(event):
         "MemberEvent": "Added {0} as collaborator".format(member_login),
         "ReleaseEvent": "Released binaries for version {0}".format(release_tag_name),
         "ForkEvent": "Forked {0}".format(repo_name),
+        "DeleteEvent": "Deleted {0} {1}".format(deleted_obj_type, deleted_obj_ref),
```
Comment (aashutoshrathi) [inline git_stalk/stalk.py]: What if we change it to
`"DeleteEvent": "Deleted {1}: {0}".format(deleted_obj_type, deleted_obj_ref.title()),`

**Comment by aashutoshrathi** [comment]
Can you check the comment once again

**Comment by merty** [comment]
Two things:

1. `deleted_obj_ref` is the name of the branch/tag already, and does not have `.title()`.
2. That would print "Deleted pointer-tracker-dep: branch" whereas my commit prints "Deleted branch: pointer-tracker-dep". Do you think we should go with that?

**Comment by aashutoshrathi** [comment]
Oh wait, your ordering is correct and by `.title()` I meant can we title case it?

**Comment by merty** [comment]
Done 👍

**Pull request merged**